app/lib/onboarding/pages/onboarding.dart

```diff
@@ -7,6 +7,39 @@ import 'package:flutter_svg/flutter_svg.dart';
 import '../../../common/module.dart';
 import '../../../common/routing/router.dart';
 
+const pagesCount = 3;
```

Comment:
```suggestion
const _pagesCount = 3;
```

Reply:
Also, I don't like having this extra variable in general. There is not a single source of truth with this in place